cmd/kubectl-gadget/utils/trace.go

```diff
@@ -68,6 +68,11 @@ type TraceConfig struct {
 	// But other gadgets, like dns, can contain output only in Started state.
 	TraceOutputState string
 
+	// TraceInitialState is the state in which the trace should be after its
+	// creation.
+	// This field is not used by *-collector gadget but needed by others.
+	TraceInitialState string
+
```

Comment:
As per my previous comment. I think this feature should be used only for multi-round gadgets. I don't think it makes sense for gadgets that make everything in a single CLI call and uses `RunTraceAndPrint*` functions.